source/agora/network/NetworkManager.d

```diff
@@ -140,9 +140,11 @@ public class NetworkManager
         // start validating and voting on the blockchain
         this.taskman.runTask(()
         {
-            while (!this.peerLimitReached())
+            while (1)
```

Comment:
Why did I change this?

Reply:
Ah yes, because this process is constant. I will add a comment.

Reply:
Walter, is that you ?